db/schema.rb

```diff
@@ -85,6 +85,8 @@
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
     t.string   "slug"
+    t.boolean  "status"
```

Comment:
do we need two columns for this?  Couldn't we have a status column that could be one of three things?  pending, approved, denied